rapid7_insightvm_cloud/Dockerfile

```
@@ -0,0 +1,26 @@
+FROM rapid7/insightconnect-python-3-38-plugin:4
```

Since we don't have any extra dependencies we should be able to use the `slim` version. This has a smaller image and uses a different base
```suggestion
FROM rapid7/insightconnect-python-3-38-slim-plugin:4
```